packages/commonwealth/client/scripts/views/pages/create_community/chain_input_rows.tsx

```diff
@@ -144,59 +144,79 @@ export const EthChainRows = (
 ) => {
   const [defaultChainNode, setDefaultChainNode] = useState<DropdownItemType>();
   const [customInputs, setCustomInputs] = useState(false);
+  const { ethChainNames, ethChains, disabled } = props;
```

Comment:
Would it be better to destructure these in line, rather than destructing within the component? 

It may be a little messy but would make things more concise.